_posts/2019-10-23-audit-openssh-security.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+current: post
+cover: openssh_article_cover 
+title: How to Audit OpenSSH Configuration and Secure It
+date: 2019-10-25 17:00:00
+category: tutorials
+author: Mahmoud Fahmy
+tags: [tutorials]
+class: post-template
+---
+
+
+## Introduction
+
+**Required Skill Level**: Beginner to Medium
+
+**Reading Time**: 15-20 Minutes
+
+In this post, we will review the fundamentals of a secure OpenSSH server. As you probably know the SSH (Secure Shell) is a method to remote login to a server. SSH is usually secure however if used with the default settings, can be vulnerable to attacks from unwanted parties. 
+
+We will look at the basics of auditing and securing your OpenSSH configuration file, to make sure that only trusted users can remote login to your server, and are privy to the information they are permitted to.
+
+**Requirements:**
+* A Linux Distribution (this guide should be good for most Distro's)
+* A server
+* The config file (usually located at /etc/ssh/ssh_config)
+
+you can open this file from the terminal using the command:
+
+~~~
```

Comment:
I don't think this is proper Markdown. See [this reference](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting)